lib/internal/async_hooks.js

```diff
@@ -194,9 +194,7 @@ function emitInitNative(asyncId, type, triggerAsyncId, resource) {
   resource = lookupPublicResource(resource);
   // Use a single try/catch for all hooks to avoid setting up one per iteration.
   try {
-    // Using var here instead of let because "for (var ...)" is faster than let.
-    // Refs: https://github.com/nodejs/node/pull/30380#issuecomment-552948364
-    for (var i = 0; i < active_hooks.array.length; i++) {
+    for (let i = 0; i < active_hooks.array.length; i++) {
```

Comment:
I would restrain from making this change unless we have ample evidence that it's as fast as.

Reply:
@mcollina the `for(var...)` loop in `os.networkInterfaces()` was changed to `for(let...)` in https://github.com/nodejs/node/pull/42563 and I ran a benchmark for that in https://github.com/nodejs/node/pull/42563#issuecomment-1085913701 and there was no significant perf regression. Could the results be different for these for loops?

Reply:
This is one of the hottest loops of the whole Node.js. Keeping a var here does not hurt anybody - most contributors should not even come to this file. Let this be and avoid future problems.

Reply:
Better to run a benchmark for this PR as well to find out if it's as fast or not. The performance edge cases were rather fixed in V8 years ago, as also seen in the mentioned PR (#42563).

Reply:
I'll use comments to disable linting for the ones @mcollina has flagged, and then we can do benchmarks on them or leave them as appropriate without holding up landing the rest of this.